Cross.toml

```
@@ -0,0 +1,24 @@
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[target.armv7-unknown-linux-musleabi]
+pre-build = [
+  # additional commands to run prior to building the package
+  "dpkg --add-architecture armhf",
+  "apt-get update && DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install crossbuild-essential-armhf cmake clang",
```

Since all other dependencies are vendored, we can add only `clang` to satisfy the bindgen requred by aws-lc-sys
```suggestion
  "apt-get update",
  "apt-get --assume-yes install clang"
```

---

I don't think cmake is not installed by default on ubuntu container, so I think it would be better to explicitly install the dependencies to prevent future breakages.

I removed the crossbuild-essential.